dev/release/post-05-ruby.sh

```diff
@@ -32,14 +32,16 @@ version=$1
 archive_name=apache-arrow-${version}
 tar_gz=${archive_name}.tar.gz
 
+source "${SOURCE_DIR}/git-vars.sh"
+
 echo "NOTE: We should release RubyGems after Homebrew and MSYS2 packages are updated!!!"
 
 echo "Checking Homebrew package..."
 homebrew_version=$(
   curl \
     --fail \
     --no-progress-meter \
-    https://raw.githubusercontent.com/Homebrew/homebrew-core/master/Formula/apache-arrow-glib.rb | \
+    https://raw.githubusercontent.com/Homebrew/homebrew-core/${DEFAULT_BRANCH}/Formula/apache-arrow-glib.rb | \
```

Comment:
Shouldn't change this one since their default branch won't necessarily be ours (and anyway if they were to change, the github http server will redirect fine)

```suggestion
    https://raw.githubusercontent.com/Homebrew/homebrew-core/master/Formula/apache-arrow-glib.rb | \
```

Reply:
Thank you, @nealrichardson, this makes sense to me. I'll also revert the change to source `git-vars.sh` in this file.